projects/anomaly-detection-stack/layers/loadgen.yml

```diff
@@ -14,16 +14,18 @@
         Name: "smart-load-gen"
       ec2_group: smart-load-gen
 
-- name: Setup instance
+- name: smart-load-gen | Install software
   hosts: smart-load-gen
   gather_facts: False
-  tags: ['aws-setup', 'aws-start']
   remote_user: ubuntu
   become: yes
   become_method: sudo
+  tags:
+    - aws-setup
+    - aws-start
   pre_tasks:
     - name: Gathering facts
       setup:
```

Comment:
Fixed in [4627034](https://github.com/smartcat-labs/smartcat-ops/commit/4627034b45a2c275f6e0c800033ef6c83bd6b53b)